NOTES-VMS.md

```diff
@@ -68,7 +63,7 @@
  directly for debugging.  Do not try to use them from a script, such
  as running the test suite.
 
- *The following is not available on Alpha*
+ `The following is not available on Alpha`
```

Comment:
Uhmmmm, why?  Italics was intended

Reply:
Oh, don't know why. Fixed.

Reply:
Meanwhile I recall why I did a change at this point:
md-nits did not like it, reporting
```
./NOTES-VMS.md:66: MD036 Emphasis used instead of a header
```

So now I've resorted to
```
### The following is not available on Alpha
```